lib/Onnxifi/Base.cpp

```diff
@@ -75,8 +73,15 @@ onnxStatus BackendId::checkGraphCompatibility(const void *onnxModel,
   return ONNXIFI_STATUS_SUCCESS;
 }
 
-void Backend::runAsync(std::function<void(void)> &&fn) {
-  threadPool_.submit(std::move(fn));
+// static
+std::unique_ptr<runtime::HostManager>
+BackendId::createHostManager(glow::BackendKind kind) {
+  std::vector<runtime::DeviceConfig> configs;
+  auto config = runtime::DeviceConfig();
```

Comment:
This will need to get updated once you rebase. It's now DeviceManagerConfig. Check #2483 
The new deviceconfig object should probably left blank unless you have device specific logic.

Reply:
Sounds good, I'll rebase and update it